src/Nest/Aggregations/Bucket/Histogram/HistogramAggregation.cs

```diff
@@ -26,11 +26,8 @@ public interface IHistogramAggregation : IBucketAggregation
 		[JsonProperty("extended_bounds")]
 		ExtendedBounds<double> ExtendedBounds { get; set; }
 
-		[JsonProperty("pre_offset")]
-		long? PreOffset { get; set; }
-
-		[JsonProperty("post_offset")]
-		long? PostOffset { get; set; }
+		[JsonProperty("offset")]
+		double? Offset { get; set; }
```

Comment:
[It looks like this can be a `Time` type](https://github.com/elastic/elasticsearch/blob/9f3a34efa34b500147f3235a31043ceab685d410/core/src/main/java/org/elasticsearch/search/aggregations/bucket/histogram/DateHistogramAggregationBuilder.java#L212-L212) for greater flexibility.

Reply:
I may be misunderstanding your request but I'm not sure this is correct. This change applies to the `HistorygramAggregation` and not to `DateHistogramAggregation`. Also in the [java counterpart `HistogramAggregationBuilder`](https://github.com/elastic/elasticsearch/blob/9f3a34efa34b500147f3235a31043ceab685d410/core/src/main/java/org/elasticsearch/search/aggregations/bucket/histogram/HistogramAggregationBuilder.java#L149) doesn't have this method. I may be missing something so could you please explain?